profiles/utils.py

```diff
@@ -65,7 +65,8 @@ def set_current_service(service):
 
 
 def get_current_user():
-    return getattr(_thread_locals, "user", None)
+    req = getattr(_thread_locals, "request", None)
```

Comment:
Could the variable name just be "request"?

Reply:
Yes. Changed within 867f60bced.